### PR TITLE
Update pom.xml

### PR DESCRIPTION
**Pull request by gabrielli696**
junit5-check please

### CODE REVIEW

File: src/erpe/pom.xml [hunk 1]
```diff
@@ -91,10 +91,25 @@
       <artifactId>slf4j-log4j12</artifactId>
     </dependency>
 
+
+   <dependency>
+       <groupid>org.junit.jupiter</groupid>
+        <artifactid>junit-jupiter-api</artifactid>
+        <version>5.0.0-M3</version>
+        <scope>test</scope>
+    </dependency>
```
Comment (masterdany88) [inline src/erpe/pom.xml:99]: What this is? Why jupiter?

File: src/erpe/pom.xml [hunk 2]
```diff
@@ -91,10 +91,25 @@
...
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <scope>test</scope>
+       <groupid>junit</groupid>
+       <artifactid>junit</artifactid>
+       <version>4.12</version>
+       <scope>test</scope>
+   </dependency>
```
Comment (masterdany88) [inline src/erpe/pom.xml:105]: Again why version 4. We will start with 5 beta version.

File: src/erpe/pom.xml [hunk 3]
```diff
@@ -91,10 +91,25 @@
...
+ 
+    <dependency>
+        <groupid>pl.pragmatists</groupid>
+        <artifactid>JUnitParams</artifactid>
+        <version>1.0.6</version>
+        <scope>test</scope>
```
Comment (masterdany88) [inline src/erpe/pom.xml:111]: Do not know what this is.

File: src/erpe/pom.xml [hunk 4]
```diff
@@ -262,6 +277,29 @@
...
+                   <version>5.0.0-M3</version>
+                </dependency>
+                <dependency>
+                    <groupid>org.junit.vintage</groupid>
+                    <artifactid>junit-vintage-engine</artifactid>
+                   <version>4.12.0-M3</version>
+                </dependency>
+           </dependencies>
+        </plugin>
+
```
Comment (masterdany88) [inline src/erpe/pom.xml:314]: Where did You get this from?

**Review by masterdany88 — CHANGES_REQUESTED** [review]
Some explanation please

File: src/erpe/pom.xml (hunk 4 above)
Comment (gabrielli696) [reply src/erpe/pom.xml:314]: All this are plugins which are necessary to use JUnit5 which isn't  official verison.
After official release there will be required changes to official librar.

File: src/erpe/pom.xml (hunk 3 above)
Comment (gabrielli696) [reply src/erpe/pom.xml:111]: Parametrised test with adnotations as input:
https://github.com/Pragmatists/JUnitParams

File: src/erpe/pom.xml (hunk 2 above)
Comment (gabrielli696) [reply src/erpe/pom.xml:105]: This is actual version we extend with Jupiter

File: src/erpe/pom.xml (hunk 1 above)
Comment (gabrielli696) [reply src/erpe/pom.xml:99]: Jupiter is engine for JUnit5

File: src/erpe/pom.xml (hunk 3 above)
Comment (masterdany88) [reply src/erpe/pom.xml:111]: We can try this. I do not know this framework, but looks good.

File: src/erpe/pom.xml (hunk 2 above)
Comment (masterdany88) [reply src/erpe/pom.xml:105]: so this is required by juint 5? I've saw junit 5 example without junit 4 dependencies.

File: src/erpe/pom.xml (hunk 1 above)
Comment (masterdany88) [reply src/erpe/pom.xml:99]: ok

**Review by masterdany88 — APPROVED** [review]

**Pull request merged**